src/app/hooks/useGa.ts

```diff
@@ -81,6 +82,12 @@ export const useGa = (): {
   return {
     gtag: {
       record: (options) => {
+        // Guard against this running server-side.
+        /* c8 ignore next 4 */
+        if (typeof window === "undefined") {
+          console.warn("window is not defined");
```

Comment:
Where toes this `console.warn()` go? User's Devtools? Or server logs?
Or more importantly, do we care, or is this just flooding logs?

Reply:
It goes to server logs, and it is an issue we want to fix if it shows up. Not enough to throw an exception and crash, but enough that we should figure out why a server component is trying to use this.